Java/app/src/main/java/com/taboola/sdk3example/sdk_native/SDKNativeMenu.java

```diff
@@ -0,0 +1,104 @@
+package com.taboola.sdk3example.sdk_native;
+
+import android.os.Bundle;
+
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.taboola.sdk3example.R;
+
+
+public class SDKNativeMenu extends AppCompatActivity   implements SDKNativeMenuFragment.OnFragmentInteractionListener {
```

Comment:
Why is our Activity called a "..Menu"?

Reply:
as parity to SDKWebMenu and SDKClassicMenu those activities with menu to choose which screen to display